tests/testthat/test_Plot_synergy.R

```diff
@@ -0,0 +1,216 @@
+library("testthat")
```

Comment:
package "testthat" should be in the namespace so no need to need to load here.

Reply:
Done.